common/tier4_calibration_rviz_plugin/src/accel_brake_map_calibrator_button_panel.cpp

```diff
@@ -118,25 +118,28 @@ void AccelBrakeMapCalibratorButtonPanel::pushCalibrationButton()
   status_label_->setText("executing calibration...");
 
   std::thread thread([this] {
-    auto req = std::make_shared<tier4_vehicle_msgs::srv::UpdateAccelBrakeMap::Request>();
-    req->path = "";
-
-    client_->async_send_request(
-      req,
-      [this](
-        [[maybe_unused]] rclcpp::Client<tier4_vehicle_msgs::srv::UpdateAccelBrakeMap>::SharedFuture
-          result) {
-        status_label_->setStyleSheet("QLabel { background-color : lightgreen;}");
-        status_label_->setText("OK!!!");
-
-        // wait 3 second
-        after_calib_ = true;
-        rclcpp::Rate(3.0).sleep();
-        after_calib_ = false;
-
-        // unlock button
-        calibration_button_->setEnabled(true);
-      });
+    if (!client_->wait_for_service(std::chrono::seconds(1))) {
+      status_label_->setStyleSheet("QLabel { background-color : red;}");
+      status_label_->setText("service server not found");
+
+    } else {
+      auto req = std::make_shared<tier4_vehicle_msgs::srv::UpdateAccelBrakeMap::Request>();
+      req->path = "";
+      client_->async_send_request(
+        req, [this]([[maybe_unused]] rclcpp::Client<
+                    tier4_vehicle_msgs::srv::UpdateAccelBrakeMap>::SharedFuture result) {});
+
+      status_label_->setStyleSheet("QLabel { background-color : lightgreen;}");
+      status_label_->setText("OK!!!");
+    }
+
+    // wait 3 second
+    after_calib_ = true;
+    rclcpp::Rate(0.33).sleep();
```

Comment:
> 0.33

Is this a mistake?

Reply:
@1222-takeshi 
This is one of the minor bug fixes.
```rclcpp::Rate(3.0).sleep()``` means 1/3 second sleep.

Reply:
This is a bit misleading, so I changed this line to ```rclcpp::Rate( 1.0 / 3.0 ).sleep();```